sdks/python/apache_beam/runners/interactive/utils_test.py

```diff
@@ -42,6 +42,30 @@
 from apache_beam.utils.timestamp import Timestamp
 from apache_beam.utils.windowed_value import WindowedValue
 
+# Protect against environments where apitools library is not available.
+try:
```

Comment:
You can skip tests when there is an ImportError.

For example: https://github.com/apache/beam/blob/70d9e2a08cc32192790cd9c98ffa15a756877a73/sdks/python/apache_beam/runners/interactive/display/pcoll_visualization_test.py#L47

And https://github.com/apache/beam/blob/master/sdks/python/apache_beam/runners/interactive/testing/integration/screen_diff.py#L129

Reply:
Thanks for the sample code, I've added this change now.